contrib/completion/zsh/_docker

```diff
@@ -138,6 +138,14 @@ __docker_containers_names() {
     __docker_get_containers all names "$@"
 }
 
+__docker_plugins() {
+    [[ $PREFIX = -* ]] && return 1
+    integer ret=1
+    plugins=(${(f)"$(_call_program commands docker $docker_options info | sed -n "/^Plugins/,/^[^ ]/s/ $1: //p")"})
```

Comment:
It would be better to avoid the use of sed since it's easy to have something that wouldn't work on OSX (like we had in the past).

Are all the plugins on the same line? If yes, `(f)` only split on new lines. Use `(ps: :)` instead. Otherwise, it should work. Also, you are missing a `local -a plugins`.

Give me an example of output of `docker info`, I'll get rid of the sed.